### DISCUSSION TITLE
What is the significance of the "linkerd-crds-chart-version" data in the `linkerd-config`

### DISCUSSION THREAD

**Discussion by gkarthiks** [linkerd/linkerd2#10740] [Q&A]
I see the `linkerd-config` ConfigMap is having a key/value for the `linkerd-crds-chart-version` [here](https://github.com/linkerd/linkerd2/blob/main/charts/linkerd-control-plane/templates/config.yaml#L14) 

Would like to know what is the significance of this key/value pair?

**Comment by alpeb** [comment]
Good question. I couldn't find any use of it. Should be safe to remove, plus its value is outdated anyways.
